src/Storages/WindowView/StorageWindowView.h

```diff
@@ -150,6 +150,7 @@ class StorageWindowView final : public shared_ptr_helper<StorageWindowView>, pub
 private:
     Poco::Logger * log;
 
+    ASTPtr select_query;
     ASTPtr mergeable_query;
     ASTPtr final_query;
```

Comment:
Please add a comment in this file about the differencies between these three variables. Now it requires to reread all sequence of visitors and modifications to remember the difference.